src/azure-cli/azure/cli/command_modules/cdn/linter_exclusions.yml

```diff
@@ -0,0 +1,109 @@
+            - option_length_too_long
+        private_link_resource:
+            rule_exclusions:
+            - option_length_too_long
+        private_link_request_message:
+            rule_exclusions:
+            - option_length_too_long
+afd origin update:
+    parameters:
+        private_link_location:
+            rule_exclusions:
+            - option_length_too_long
+        private_link_sub_resource_type:
+            rule_exclusions:
+            - option_length_too_long
+        private_link_resource:
+            rule_exclusions:
+            - option_length_too_long
+        private_link_request_message:
+            rule_exclusions:
+            - option_length_too_long
+afd origin-group create:
+    parameters:
+        load_balancing_additional_latency_in_milliseconds:
+            rule_exclusions:
+            - option_length_too_long
+        load_balancing_sample_size:
+            rule_exclusions:
+            - option_length_too_long
+        load_balancing_successful_samples_required:
+            rule_exclusions:
+            - option_length_too_long
+        probe_interval_in_seconds:
+            rule_exclusions:
+            - option_length_too_long
+afd origin-group update:
+    parameters:
+        load_balancing_additional_latency_in_milliseconds:
+            rule_exclusions:
+            - option_length_too_long
+        load_balancing_sample_size:
+            rule_exclusions:
+            - option_length_too_long
+        load_balancing_successful_samples_required:
+            rule_exclusions:
+            - option_length_too_long
+        probe_interval_in_seconds:
+            rule_exclusions:
+            - option_length_too_long
+afd route create:
+    parameters:
+        content_types_to_compress:
+            rule_exclusions:
+            - option_length_too_long
+        link_to_default_domain:
+            rule_exclusions:
+            - option_length_too_long
+        query_string_caching_behavior:
+            rule_exclusions:
+            - option_length_too_long
+afd route update:
+    parameters:
+        content_types_to_compress:
+            rule_exclusions:
+            - option_length_too_long
+        link_to_default_domain:
+            rule_exclusions:
+            - option_length_too_long
+        query_string_caching_behavior:
+            rule_exclusions:
+            - option_length_too_long
+afd rule create:
+    parameters:
+        preserve_unmatched_path:
+            rule_exclusions:
+            - option_length_too_long
+        query_string_behavior:
+            rule_exclusions:
+            - option_length_too_long
+        match_processing_behavior:
+            rule_exclusions:
+            - option_length_too_long
+afd rule action add:
+    parameters:
+        preserve_unmatched_path:
+            rule_exclusions:
+            - option_length_too_long
+        query_string_behavior:
+            rule_exclusions:
+            - option_length_too_long
```

Comment:
Can you add short name for them?

Reply:
Trip the leading "load_balancing" for those load balancing related ones.

I could not find a suitable short name for other parameters, let me know if you have any ideas? :)